### PR TITLE
[board] Nucleo-F767ZI support

### PR DESCRIPTION
**Pull request by rleh**
- [x] It compiles
- [x] Verify clock config (CubeMX)
- [x] Update README 🤦🏽‍♀️ 
- [ ] Tested on hardware

### CODE REVIEW

File: src/modm/board/nucleo_f767zi/board.hpp [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+#include <modm/platform.hpp>
+#include <modm/architecture/interface/clock.hpp>
+#include <modm/debug/logger.hpp>
+/// @ingroup modm_board_nucleo_F767ZI
+#define MODM_BOARD_HAS_LOGGER
+
+using namespace modm::platform;
+
+/// @ingroup modm_board_nucleo_F767ZI
```
Comment (salkinium) [inline src/modm/board/nucleo_f767zi/board.hpp]: FYI: The `:docs` generator expects this to be lower case, not sure if Doxypress is case-sensitive and would link this correctly.

**Comment by chris-durand** [comment]
Should we add USB support for this board? We didn't test USB on any F7 so far. Should be quite trivial to do. I could have a look at it later.

**Comment by salkinium** [comment]
I tested it on the F7 Disco board, but could only get the low speed interface working. Not sure why the HS interface didn't work…

**Review by salkinium — APPROVED** [review]
LGTM so far, but I don't have the hardware to test this on.

**Comment by salkinium** [comment]
Btw, since this is a Cortex-M7 it may be useful to enable the Data-Cache at some point: #485.

**Comment by mikewolfram** [comment]
The pull request #466 already introduced this board long time ago. But we never got it to the end. That one also includes the example how to use ethernet.

BTW; The board.hpp in this PR looks pretty much like a copy of mine. ;-)

**Comment by salkinium** [comment]
But… don't we want to merge this one instead? The other PR is still a draft.

**Comment by rleh** [comment]
I currently dont't have the Nucleo board available for testing and #466 isn't a draft anymore, so let's merge #466.

**Pull request closed without merging**